resources/assets/lib/interfaces/current-user.ts

```diff
@@ -8,6 +8,7 @@ import GameMode from './game-mode';
 
 export default interface CurrentUser extends UserJsonExtended {
   blocks: UserRelationJson[];
+  follow_user_modding: number[];
```

Comment:
I wonder if nested object would be more useful for this, but then again we don't know if more `follow` ids would go in there 🤔